templates/index.html

```diff
@@ -1,481 +1,508 @@
+              {{ image (
+              url="https://assets.ubuntu.com/v1/9899746b-kafka-logo.png",
               alt="",
-              width="410",
-              height="220",
+              width="150",
+              height="205",
               hi_def=True,
-            ) | safe
-          }}
+              loading="auto|lazy"
+              ) | safe
+              }}
+            </div>
+          </div>
         </div>
       </div>
-    </div>
-    <div class="col-8">
-      <h2>Open source data operations</h2>
-      <p class="p-heading--4 u-sv2">Composable, model-driven solutions</p>
-      <p>Make use of open source integration code and compound the benefit of <a href="https://charmhub.io/">300+ available Juju charm operators</a>.</p>
-      <p>The Juju charm operator solution from Canonical makes deploying and managing Charmed Kubeflow predictable, repeatable and reliable for your IT operations team. Zero friction, from proof-of-concept to production.</p>
-      <p>Engineered for flexibility around complex scenarios, the Juju charms extended ecosystem allows you to tailor Charmed Kubeflow to your needs with additional integrations &mdash; like MLFlow, Apache Spark and Seldon Core.</p>
-      <p>
-        <a href="https://jaas.ai/kubeflow/bundle" class="p-button--neurtal">Kubeflow charms and models&nbsp;&rsaquo;</a>
-      </p>
-    </div>
-  </div>
-</section>
-
-<section class="p-strip--light">
-  <div class="row">
-    <div class="col-7">
-      <h2>Optimised for every cloud</h2>
-      <p class="p-heading--4 u-sv2">Fast AI predictions on every major cloud with GPU acceleration</p>
-      <p>We work with Amazon AWS, Microsoft Azure, Google and Oracle clouds to simplify multi-cloud, GPU accelerated AI.</p>
-      <p>Easy setup across cloud Kubernetes services for a multi-cloud MLOps platform. Get the elasticity of the public clouds with the governance of on-premise service management. Separate training from production environments as required &mdash; hybrid cloud is also supported.</p>
-      <p>Automated deployment and management with Canonical's Juju charm model-driven operator solution. Support for major cloud Kubernetes services including Azure AKS, AWS EKS and Google GKE.</p>
-      <p><a href="https://ubuntu.com/kubernetes">Kubeflow on multi-cloud K8s&nbsp;&rsaquo;</a></p>
-    </div>
-    <div class="col-5 u-vertically-center p-logo-section">
-      <div class="p-logo-section__items" style="margin: .5rem">
-        <div class="p-logo-section__item" style="margin: 1.875rem">
-          {{
-            image(
-              url="https://assets.ubuntu.com/v1/e7a3c4d8-MicrosoftAzure.svg",
-              alt="Microsoft Azure",
-              width="174",
-              height="50",
-              hi_def=True,
-              attrs={"class": "p-logo-section__logo"},
-            ) | safe
-          }}
+    </section>
+    <section class="p-strip">
+      <div class="row">
+        <div class="col-4 u-hide--small u-hide--medium ">
```

Comment:
```suggestion
        <div class="col-4 u-hide--small u-hide--medium u-vertically-align">
```
Just to match the other logos